src/Controller/EventsController.php

```diff
@@ -747,7 +748,8 @@ public function approveAction(Request $request, PDO $db): void
         }
 
         // Send a notification email as we have approved
-        $event        = $event_mapper->getEventById($event_id, true)['events'][0];
+        $event = $event_mapper->getEventById($event_id, true);
```

Comment:
I think we can merge these back into one assignment now that we either return or throw, so we don't immediately overwrite the var? Same in L795